jumanji/wrappers.py

```diff
@@ -360,18 +360,40 @@ def render(self, state: State) -> Any:
         return super().render(state_0)
 
 
+OBS_IN_EXTRAS_KEY = "real_next_obs"
```

Comment:
I think `NEXT_OBS_KEY_IN_EXTRAS` is probably preferred, and `"real_next_obs"` is ambiguous (because of "real"). What do you think of the following?
```suggestion
NEXT_OBS_KEY_IN_EXTRAS = "next_obs"
```